ghostTemplate.yaml

```diff
@@ -0,0 +1,381 @@
+            - Name: database__pool__min
+              Value: !If [GhostAuroraDB, '0', !Ref "AWS::NoValue"]
+            - Name: database__connection__port
+              Value: !If [GhostAuroraDB, '3306', !Ref "AWS::NoValue"]
+            - Name: database__connection__database
+              Value: !If [GhostAuroraDB, !Ref ghostDB, !Ref "AWS::NoValue"]
+            # string version of the Amazon Root CA 1 trust store which Ghost uses for SSL to Aurora MySQL
+            # https://www.amazontrust.com/repository/AmazonRootCA1.pem
+            # https://ghost.org/docs/concepts/config/
+            - Name: database__connection__ssl__ca
+              Value: !If [GhostAuroraDB, "-----BEGIN CERTIFICATE-----\nMIIDQTCCAimgAwIBAgITBmyfz5m/jAo54vB4ikPmljZbyjANBgkqhkiG9w0BAQsF\nADA5MQswCQYDVQQGEwJVUzEPMA0GA1UEChMGQW1hem9uMRkwFwYDVQQDExBBbWF6\nb24gUm9vdCBDQSAxMB4XDTE1MDUyNjAwMDAwMFoXDTM4MDExNzAwMDAwMFowOTEL\nMAkGA1UEBhMCVVMxDzANBgNVBAoTBkFtYXpvbjEZMBcGA1UEAxMQQW1hem9uIFJv\nb3QgQ0EgMTCCASIwDQYJKoZIhvcNAQEBBQADggEPADCCAQoCggEBALJ4gHHKeNXj\nca9HgFB0fW7Y14h29Jlo91ghYPl0hAEvrAIthtOgQ3pOsqTQNroBvo3bSMgHFzZM\n9O6II8c+6zf1tRn4SWiw3te5djgdYZ6k/oI2peVKVuRF4fn9tBb6dNqcmzU5L/qw\nIFAGbHrQgLKm+a/sRxmPUDgH3KKHOVj4utWp+UhnMJbulHheb4mjUcAwhmahRWa6\nVOujw5H5SNz/0egwLX0tdHA114gk957EWW67c4cX8jJGKLhD+rcdqsq08p8kDi1L\n93FcXmn/6pUCyziKrlA4b9v7LWIbxcceVOF34GfID5yHI9Y/QCB/IIDEgEw+OyQm\njgSubJrIqg0CAwEAAaNCMEAwDwYDVR0TAQH/BAUwAwEB/zAOBgNVHQ8BAf8EBAMC\nAYYwHQYDVR0OBBYEFIQYzIU07LwMlJQuCFmcx7IQTgoIMA0GCSqGSIb3DQEBCwUA\nA4IBAQCY8jdaQZChGsV2USggNiMOruYou6r4lK5IpDB/G/wkjUu0yKGX9rbxenDI\nU5PMCCjjmCXPI6T53iHTfIUJrU6adTrCC2qJeHZERxhlbI1Bjjt/msv0tadQ1wUs\nN+gDS63pYaACbvXy8MWy7Vu33PqUXHeeE6V/Uq2V8viTO96LXFvKWlJbYK8U90vv\no/ufQJVtMVT8QtPHRh8jrdkPSHCa2XV4cdFyQzR1bldZwgJcJmApzyMZFo6IQ6XU\n5MsI+yMRQ+hDKXJioaldXgjUkK642M4UwtBV8ob2xJNDd2ZhwLnoQdeXeGADbkpy\nrqXRfboQnoZsG4q5WTP468SQvvG5\n-----END CERTIFICATE-----\n", !Ref "AWS::NoValue"]
+      Cpu: '256'
+      ExecutionRoleArn: !GetAtt ghostAPIExecutionRole.Arn
+      Memory: '512'
+      NetworkMode: awsvpc
+      RequiresCompatibilities:
+        - FARGATE
+      TaskRoleArn: !If [GhostAuroraDB, !GetAtt ghostAPITaskRoleAurora.Arn, !GetAtt ghostAPITaskRoleEFS.Arn]
+      Volumes: 
+        - Name: ghostVolume
+          EFSVolumeConfiguration:
+            FilesystemId: !GetAtt ghostFS.FileSystemId
+            AuthorizationConfig:
+              IAM: "ENABLED"
+            TransitEncryption: 'ENABLED'
+    DependsOn: ghostAPILogGroup
+  Service:
+    Type: AWS::ECS::Service
+    Properties:
+      ServiceName: !Sub ghostAPIservice-${EnvironmentTagName}
+      PlatformVersion: 1.4.0
+      Cluster: default
+      TaskDefinition: !Ref ghostAPI
+      DesiredCount: 2
+      DeploymentConfiguration:
+        MaximumPercent: 200
+        MinimumHealthyPercent: 100
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: ENABLED
+          Subnets: !Ref DefaultVPCSubnets
```

Comment:
We fixed the EFS mountpoints by selecting only the first two subnets of DefaultVPCSubnets (because we don't know how many there will be). But if there are more than two subnets (which is common among all the regions), this is what will cause the service to take a while to spin up and become healthy if ECS starts containers in subnets without an EFS mount point (at least, this is what you and I both believe is the case based on the error message you shared in a comment).

What we can do is modify this (and other references to DefaultVPCSubnets) to be:
```yaml
Subnets:
  - !Select 
    - 0
    - !Ref DefaultVPCSubnets
  - !Select 
    - 1
    - !Ref DefaultVPCSubnets
```

We're venturing into more and more manual stack manipulation, which I don't love, but the EFS mount point CFN specification is annoyingly making us do so. Why couldn't they be like every other resource and just let you specify one "mount point" and provide a list of subnets to connect to?